src/prefect/utilities/graphql.py

```diff
@@ -45,6 +45,19 @@ def __str__(self) -> str:
         return self.value
 
 
+def LiteralSetValue(value: list) -> str:
+    """
+    When parsing GraphQL arguments, strings can be wrapped in this class to be rendered
```

Comment:
Not actually a class; maybe:
> When parsing GraphQL arguments, `LiteralSetValue` renders strings as literal set values...

Not really a big deal since this isn't truly a public API

Reply:
Ah thanks, I copied this from `EnumValue` because originally i implemented this as a class but oddly enough our parser has an easier time if it's just a prebuilt string.